app/helpers/custom_form_error_helper.rb

```diff
@@ -0,0 +1,19 @@
+module CustomFormErrorHelper
+  def errors(object, attribute)
+    error_messages = object.try(:errors).try(:full_messages_for, attribute)
+
+    if error_messages.is_a?(Array) and error_messages.count > 0
+      error_contents = create_error_div(error_messages)
+    end
+    error_contents || ''
+  end
+
+  def create_error_div(error_messages)
+    result = <<~HTML
+      <small class="help-block">#{error_messages.first}</small>
+    HTML
+    # rubocop:disable: Rails/OutputSafety
+    raw result
```

Comment:
__[RuboCop]__ Rails/OutputSafety: Tagging a string as html safe may be a security risk. ([link](https://sider.review/gh/repos/47168038/pulls/585#issue-4232501))

<sub>posted by Sider</sub>